Source/JavaScriptCore/Sources.txt

```diff
@@ -46,17 +46,18 @@ API/OpaqueJSString.cpp
 
 assembler/AbstractMacroAssembler.cpp
 assembler/AssemblerBuffer.cpp
+assembler/AssemblyComments.cpp
 assembler/CPU.cpp
 assembler/JITOperationList.cpp
 assembler/LinkBuffer.cpp
 assembler/MacroAssembler.cpp
-assembler/MacroAssemblerARM64.cpp
-assembler/MacroAssemblerARMv7.cpp
+assembler/MacroAssemblerARM64.cpp @no-unify
```

Comment:
Is this necessary?  Ditto for the rest below?  Isn't it only MacroAssemblerARMv7 that has the aliasing issue?

Reply:
Yes, but I believe they all have the same typedef. I haven't proven that this could cause an issue, but I think these files might be big enough anyway?